std/encoding/mod.ts

```diff
@@ -0,0 +1,14 @@
+export {
+  CsvHeaderOptions,
+  CsvParseError,
+  ParseCsvOptions,
+  parseCsv
+} from "./csv.ts";
+export {
```

Comment:
Why can't we do a namespace export?

Reply:
Hiding 'package' private exports which I've presumed are there for testing.

Reply:
hmmm... makes me question the testing then! 😉 (not your fault, just is a sniff that someone is breaking encapsulation then if they have exports only for testing)